sources/rest_api/typing.py

```diff
@@ -254,6 +259,10 @@ class ResourceBase(TypedDict, total=False):
     table_format: Optional[TTableHintTemplate[TTableFormat]]
     selected: Optional[bool]
     parallelized: Optional[bool]
+    processing_steps: Optional[List[ProcessingSteps]]
+    # row_filter: Optional[Callable[[Any], bool]]
+    # transform: Optional[Callable[[Any], Any]]
+    # exclude_columns: Optional[List[jsonpath.TJsonPath]]
```

Comment:
Let's remove the commented code and implement these ready-to-use operation as a follow-up PR